src/components/filters/time-widget.js

```diff
@@ -103,9 +102,7 @@ const StyledTitle = styled(CenterFlexbox)`
   }
 `;
 
-TimeWidgetFactory.deps = [SpeedControlFactory, TimeRangeFilterFactory, FloatingTimeDisplayFactory];
-
-function TimeWidgetFactory(SpeedControl, TimeRangeFilter, FloatingTimeDisplay) {
```

Comment:
put the deps definition on top of the function declaration for easy read.

Reply:
Done